apps/src/lab2/views/components/workspace-header.module.scss

```
@@ -0,0 +1,5 @@
+@import '../../../mixins.scss';
```

`@use` instead of `@import` is now preferred (see [docs](https://sass-lang.com/documentation/at-rules/use/#differences-from-import)):

```suggestion
@use '../../../mixins.scss' as mixins;

.centerHeaderContent {
  @include mixins.center-header-content(true);
}
```

---

Ooh nice. Updated, thanks!